api/src/test/java/org/openmrs/api/VisitServiceTest.java

```diff
@@ -789,6 +777,49 @@ public void purgeVisit_shouldFailIfTheVisitHasEncountersAssociatedToIt() {
 		Context.getVisitService().purgeVisit(visit);
 	}
 	
+	/**
+	 * @see VisitService#purgeVisit(Visit)
+	 */
+	@Test
+	public void purgeVisit_shouldEraseTheVisitFromTheDatabase() {
+		VisitService vs = Context.getVisitService();
+		Integer originalSize = vs.getVisits(null, null, null, null, null, null, null, null, null, true, true).size();
+		Visit visit = Context.getVisitService().getVisit(1);
+		vs.purgeVisit(visit);
+		assertEquals(originalSize - 1, vs.getVisits(null, null, null, null, null, null, null, null, null, true, true).size());
+	}
+	
+	/**
+	 * @see VisitService#purgeVisit(Visit)
+	 */
+	@Test
+	public void purgeVisit_shouldReturnWithoutVoidingIfVisitIsUnsaved () {
+		VisitService vs = Context.getVisitService();
+		Integer originalSize = vs.getVisits(null, null, null, null, null, null, null, null, null, true, true).size();
+		vs.purgeVisit(new Visit(  ));
+		assertEquals( (long)originalSize, vs.getVisits(null, null, null,null, null, null, null, null, null, true, true).size());
+	}
+	
+	/**
+	 * @see VisitService#purgeVisit(Visit)
+	 */
+	@Test(expected = APIException.class)
+	public void purgeVisit_shouldFailIfVisitHasActiveEncounters() {
```

Comment:
delete this test, you created a duplicate test of `purgeVisit_shouldFailIfTheVisitHasEncountersAssociatedToIt`